src/graphql/Errors/GqlErrorHandler.php

```diff
@@ -40,7 +40,7 @@ public static function handle(array $errors, ?callable $formatter, ?UserRequestC
           $args = $previous->getArgs() ?? [];
         }
 
-        $logContext = new LogContext($userContext?->getTenantId(), $userContext?->getUserId());
+        $logContext = new LogContext($userContext);
```

Comment:
LogContext is duplicating the behavior of $userContext for the same behavior, we can merge them to avoid issues with empty data.
By using $userContext, we know the data is already validated from the beginning.